engine/src/main/resources/assets/shaders/chunk_frag.glsl

```diff
@@ -206,11 +202,12 @@ void main() {
     // ...and finally the occlusion value
     float occlusionValue = expOccValue(gl_TexCoord[1].z);
 
-    vec3 blocklightColorValue = calcBlocklightColor(blocklightValue
+    float blocklightColorBrightness = calcBlocklightColorBrightness(blocklightValue
```

Comment:
Capture just the brightness value because we can calculate the color in a different shader without using up 3 float spots.